### PR TITLE
New Library Perspective (new authoring)

### PR DESCRIPTION
**Pull request by ederign**

### CODE REVIEW

**Comment by manstis** [comment]
I notice you're not making similar changes to the "Product" Home Page; is this deliberate or an oversight?

**Comment by ederign** [comment]
@manstis I didn't put that in our "product" version because is a "work in progress" feature and there is a good chance that will be integrated with our current authoring.

**Comment by porcelli** [comment]
+1
I understand @manstis concerns, but I +1 @ederign approach to incremental changes until mature to change abroad.

MY HOPE IS THAT WE CAN KILL THE PRODUCT PAGE!!!111

File: kie-drools-wb/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/client/perspectives/DroolsAuthoringPerspective.java [hunk 1]
```diff
@@ -155,4 +156,10 @@ public void execute() {
                 .build();
     }
 
+    public void onLibraryContextSwitchEvent( @Observes final LibraryContextSwitchEvent event ) {
```
Comment (ederign) [inline kie-drools-wb/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/client/perspectives/DroolsAuthoringPerspective.java]: done.

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by ederign** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Jenkins please retest this.

**Pull request merged**